### ISSUE TITLE
rake db:test:prepare fails silently if psql is missing

### ISSUE THREAD

**Issue by fgarces** [rails/rails#17666]
If the ```psql``` command is missing, no errors are thrown when running db:test:prepare . The relevant lines from ```activerecord/lib/active_record/tasks/postgresql_database_tasks.rb``` are presented below:

```
def structure_load(filename)
    set_psql_env
    Kernel.system("psql -q -f #{Shellwords.escape(filename)} #{configuration['database']}")
end
```

**Comment by maclover7** [comment]
This should be closed since #17885 solves this issue, right?

**Comment by senny** [comment]
@maclover7 good catch. Thank you.
